docs/data/flink_connectors.yml

```diff
@@ -93,8 +93,8 @@ rabbitmq:
   compatibility: ["1.16.x", "1.17.x"]
 
 mongodb:
-  name: "Apache Flink MongoDB Connector 1.0.2"
-  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-mongodb-1.0.2/flink-connector-mongodb-1.0.2-src.tgz"
-  source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-mongodb-1.0.2/flink-connector-mongodb-1.0.2-src.tgz.asc"
-  source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-mongodb-1.0.2/flink-connector-mongodb-1.0.2-src.tgz.sha512"
-  compatibility: ["1.16.x", "1.17.x"]
+  name: "Apache Flink MongoDB Connector 1.1.0"
+  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-mongodb-1.1.0/flink-connector-mongodb-1.1.0-src.tgz"
+  source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-mongodb-1.1.0/flink-connector-mongodb-1.1.0-src.tgz.asc"
+  source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-mongodb-1.1.0/flink-connector-mongodb-1.1.0-src.tgz.sha512"
+  compatibility: ["1.17.x", "1.18.x"]
```

Comment:
The MongoDB connector version 1.1.0 is still compatible with Flink 1.16, shall we add it?

Reply:
We only need to ensure two flink minor versions according external connector wiki, we do not need promise for older flink version compatibility even it's still compatible.